gateway-server/src/main/java/org/apache/knox/gateway/config/impl/GatewayConfigImpl.java

```diff
@@ -293,6 +293,14 @@ public class GatewayConfigImpl extends Configuration implements GatewayConfig {
   private static final String GATEWAY_DATABASE_VERIFY_SERVER_CERT =  GATEWAY_CONFIG_FILE_PREFIX + ".database.ssl.verify.server.cert";
   private static final String GATEWAY_DATABASE_TRUSTSTORE_FILE =  GATEWAY_CONFIG_FILE_PREFIX + ".database.ssl.truststore.file";
 
+  // Concurrent session properties
+  private static final String GATEWAY_PRIVILEGED_USER_CONCURRENT_SESSION_LIMIT =  GATEWAY_CONFIG_FILE_PREFIX + ".privileged.user.concurrent.session.limit";
+  private static final String GATEWAY_NON_PRIVILEGED_USER_CONCURRENT_SESSION_LIMIT =  GATEWAY_CONFIG_FILE_PREFIX + ".non.privileged.user.concurrent.session.limit";
+  private static final int GATEWAY_PRIVILEGED_USER_CONCURRENT_SESSION_LIMIT_DEFAULT = 3;
+  private static final int GATEWAY_NON_PRIVILEGED_USER_CONCURRENT_SESSION_LIMIT_DEFAULT = 2;
+  private static final String GATEWAY_PRIVILEGED_USERS = GATEWAY_CONFIG_FILE_PREFIX + ".privileged.users";
+  private static final String GATEWAY_NON_PRIVILEGED_USERS = GATEWAY_CONFIG_FILE_PREFIX + ".non.privileged.users";
```

Comment:
nit: `non.priviledged.user` can be a static variable (using the previous static variable) :)

Reply:
Changed it.